src/frontend/screens/Settings/MapSettings/BackgroundMapInfo.tsx

```diff
@@ -49,77 +61,77 @@ const m = defineMessages({
-    defaultMessage: "Local highway and crossing details",
-  },
-  lvlOfDetail20: {
-    id: "screens.Settings.MapSettings.BackgroundMapInfo.lvlOfDetail20",
-    defaultMessage: "A mid-sized building",
-  },
+  // lvlOfDetail0to1: {
+  //   id: "screens.Settings.MapSettings.BackgroundMapInfo.lvlOfDetail0to1",
+  //   defaultMessage: "Whole world",
+  // },
+  // lvlOfDetail2: {
+  //   id: "screens.Settings.MapSettings.BackgroundMapInfo.lvlOfDetail2",
+  //   defaultMessage: "Subcontinental area ",
+  // },
+  // lvlOfDetail3to4: {
+  //   id: "screens.Settings.MapSettings.BackgroundMapInfo.lvlOfDetail3to4",
+  //   defaultMessage: "Largest country",
+  // },
+  // lvlOfDetail5: {
+  //   id: "screens.Settings.MapSettings.BackgroundMapInfo.lvlOfDetail5",
+  //   defaultMessage: "Large African country",
+  // },
+  // lvlOfDetail6: {
+  //   id: "screens.Settings.MapSettings.BackgroundMapInfo.lvlOfDetail6",
+  //   defaultMessage: "Large European country",
+  // },
+  // lvlOfDetail7to8: {
+  //   id: "screens.Settings.MapSettings.BackgroundMapInfo.lvlOfDetail7to8",
+  //   defaultMessage: "Small country, US state",
+  // },
+  // lvlOfDetail9: {
+  //   id: "screens.Settings.MapSettings.BackgroundMapInfo.lvlOfDetail9",
+  //   defaultMessage: "Wide area, large metropolitan area",
+  // },
+  // lvlOfDetail10: {
+  //   id: "screens.Settings.MapSettings.BackgroundMapInfo.lvlOfDetail10",
+  //   defaultMessage: "Metropolitan area",
+  // },
+  // lvlOfDetail11: {
+  //   id: "screens.Settings.MapSettings.BackgroundMapInfo.lvlOfDetail11",
+  //   defaultMessage: "City",
+  // },
+  // lvlOfDetail12: {
+  //   id: "screens.Settings.MapSettings.BackgroundMapInfo.lvlOfDetail12",
+  //   defaultMessage: "Town, or city district",
+  // },
+  // lvlOfDetail13to14: {
+  //   id: "screens.Settings.MapSettings.BackgroundMapInfo.lvlOfDetail13to14",
+  //   defaultMessage: "Village, or suburb",
+  // },
+  // lvlOfDetail15: {
+  //   id: "screens.Settings.MapSettings.BackgroundMapInfo.lvlOfDetail15",
+  //   defaultMessage: "Small road",
+  // },
+  // lvlOfDetail16: {
+  //   id: "screens.Settings.MapSettings.BackgroundMapInfo.lvlOfDetail16",
+  //   defaultMessage: "Street",
+  // },
+  // lvlOfDetail17: {
+  //   id: "screens.Settings.MapSettings.BackgroundMapInfo.lvlOfDetail17",
+  //   defaultMessage: "Block, park, addresses",
+  // },
+  // lvlOfDetail18: {
+  //   id: "screens.Settings.MapSettings.BackgroundMapInfo.lvlOfDetail18",
+  //   defaultMessage: "Some buildings, trees",
+  // },
+  // lvlOfDetail19: {
+  //   id: "screens.Settings.MapSettings.BackgroundMapInfo.lvlOfDetail19",
+  //   defaultMessage: "Local highway and crossing details",
+  // },
+  // lvlOfDetail20: {
+  //   id: "screens.Settings.MapSettings.BackgroundMapInfo.lvlOfDetail20",
+  //   defaultMessage: "A mid-sized building",
+  // },
 });
 
-type Zoom = "loading" | number | null;
+type Zoom = (number | undefined)[] | "loading";
```

Comment:
think adding `"loading"` as a type here (which is then used in React state) is kind of an odd pattern. the loading can probably be derived based on a value that is more uniform, such as `[number, number] | null`